docs/en/operation/StressBench.md

```diff
@@ -361,17 +403,26 @@ Parameters:
 
 #### `RegisterWorkerBench` 
 
-For the `RegisterWorker` RPC and the `RequestRegisterLease` RPC.
+Simulates workload for the `RegisterWorker` RPC.
 
-This benchmark calls the RPC only once.
+This benchmark simulates concurrent worker registration using the unary RPC implementation (default 
+before Alluxio 2.7).
+You may use this command to simulate the pressure when the workers start at once at your scale.
 
-This benchmark measures the performance of the process of new worker registration. 
-The`RegisterWorker` RPC carries a list of blocks that is currently stored in the worker's storage. 
-This RPC can be expensive if the list is huge. 
-The `RequestRegisterLease` RPC is part of the mechanism that the master uses to 
-control the number of concurrently registering clients so that it does not exceed the master's 
+The`RegisterWorker` RPC carries a list of blocks that is currently stored in the worker's storage.
+This RPC can be expensive if the list is huge.
+The `RequestRegisterLease` RPC is part of the mechanism that the master uses to
+control the number of concurrently registering workers so that it does not exceed the master's
 processing capacity.
 
+Note that Alluxio 2.7 introduced the [register lease]({{'/en/reference/Properties-List.html' | relativize_url }}#alluxio.master.worker.register.lease.enabled)
```

Comment:
Use this link https://docs.alluxio.io/os/user/stable/en/operation/Scalability-Tuning.html#worker-register-lease